scripts/disk_info

```diff
@@ -120,12 +120,16 @@ def filteredDiskInfo(deviceid):
         except KeyError:  # not all objects have all these attributes
             pass
 
-    # Set sizes to string with .5GB precision
+    # Set sizes to string with .5GB precision. Add filter for machines running > 10.15
+    # as these do not use the "FreeSpace" key but rather need to calculate from container.
     if out.get('TotalSize'):
-        out['TotalSize'] = str(_HalfGB(out['TotalSize']));
+        out['TotalSize'] = str(_HalfGB(out['TotalSize']))
 
-    if out.get('FreeSpace'):
-        out['FreeSpace'] = str(_HalfGB(out['FreeSpace']));
+    if _OSVersion() <= "14":
```

Comment:
Would it be possible to check the existence of `APFSContainerFree` instead of _OSVersion()? That would seem more solid to me.

Reply:
Yes - the FreeSpace key exists on both output styles, so I can switch the order of evaluation and skip the OS compare. Update incoming...